added floors

kseniia-s opened the pull request.

kseniia-s committed:
added floors

VinShurikk commented on src/Area.java:
```diff
@@ -0,0 +1,34 @@
+    return square;
+  }
+
+  //  public static final String BEDROOM = "badroom";
```
Remove commented strings

kseniia-s replied:
removed

VinShurikk commented on src/Floor.java:
```diff
@@ -0,0 +1,50 @@
+  private double nonLivingArea;
+  private double totalArea;
+
+  List<Area> areas = new ArrayList<>();
```
private field

kseniia-s replied:
what private field?

kseniia-s replied:
I got it. Fixed

VinShurikk commented on src/House.java:
```diff
@@ -0,0 +1,65 @@
+import java.util.List;
+
+public class House {
+  List<Floor> floors = new ArrayList<>();
```
private field

kseniia-s replied:
what private field?

kseniia-s replied:
I got it. Fixed

VinShurikk replied:
floors should be private

VinShurikk commented on src/House.java:
```diff
@@ -0,0 +1,65 @@
+    //вычислить на каждом этаже сколько ванных комнат
+    for (int i = 0; i < floors.size(); i++){
+      Floor floor = floors.get(i);
+      List<Area> floorAreas = floor.areas;
```
reuse already computed value.
floor.getBedroomCount();

kseniia-s replied:
corrected

VinShurikk commented on src/House.java:
```diff
@@ -0,0 +1,65 @@
+    double houseLivingArea = 0.0;
+    for (int i = 0; i < floors.size(); i++){
+      Floor floor = floors.get(i);
+      List<Area> floorAreas = floor.areas;
```
reuse already computed value.
floor.getLivingArea();

kseniia-s replied:
fixed

VinShurikk commented on src/House.java:
```diff
@@ -0,0 +1,65 @@
+    double houseNonLivingArea = 0.0;
+    for (int i = 0; i < floors.size(); i++){
+      Floor floor = floors.get(i);
+      List<Area> floorAreas = floor.areas;
```
reuse already computed value.
floor.getNonLivingArea();

kseniia-s replied:
fixed

kseniia-s committed:
Remarks resolved:
- Comments are deleted;
- Changed type of fields to private;
- Used already computed values.
